internal/provisioner/utility_group.go

```diff
@@ -67,7 +67,7 @@ func newUtilityGroupHandle(kops *kops.Cmd, provisioner *KopsProvisioner, cluster
 		return nil, errors.Wrap(err, "failed to get handle for Prometheus")
```

Comment:
@gigawhitlocks Don't we need a desiredVersion setup for Prometheus as well?

Reply:
Prometheus needs to have the Cluster object available to it so instead of passing the desired version as an argument, the Prometheus handle just gets that value from the Cluster object that it holds, here:
https://github.com/mattermost/mattermost-cloud/pull/152/files/2f28abd6277722ae6b376f0a932b4ad206e5d2a4#diff-5c338774e76a14c1ce511476e9a3f8a6R47